prow/plugins/lgtm/lgtm.go

```diff
@@ -204,3 +207,34 @@ func handle(gc githubClient, log *logrus.Entry, e *event) error {
 	}
 	return nil
 }
+
+type ghLabelClient interface {
+	RemoveLabel(owner, repo string, number int, label string) error
+}
+
+func handlePullRequest(gc ghLabelClient, pe github.PullRequestEvent) error {
+	if pe.PullRequest.Merged {
+		return nil
+	}
+
+	if pe.Action != "synchronize" {
+		return nil
+	}
+
+	// Don't bother checking if it has the label...it's a race, and we'll have
```

Comment:
We should consider checking if it has the label only to save the API token that comes with attempting to remove it.